app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  before_save { self.email = email.downcase }
```

Comment:
```suggestion
  before_save {email.downcase!}
```
trước } và sau { không cách spaces nhé em